veracruz-utils/src/csr.rs

```diff
@@ -227,7 +227,8 @@ pub fn generate_csr(
     template: &CsrTemplate,
     private_key: &EcdsaKeyPair,
 ) -> Result<Vec<u8>, CertError> {
-    let public_key = private_key.public_key().as_ref().clone();
+    #[allow(clippy::clone_double_ref)]
+    let public_key =  private_key.public_key().as_ref().clone();
```

Comment:
There seems to be an extra space added after the '=' here.

Reply:
yeah I accidentally did that. I will fix it with fixing the `clippy` attribute [here](https://github.com/veracruz-project/veracruz/pull/419#discussion_r860881874)